sonar-scanner-cli/content.md

```diff
@@ -0,0 +1,21 @@
+# What is SonarScanner CLI?
+
+[SonarScanner](https://redirect.sonarsource.com/doc/install-configure-scanner.html) is the official scanner used to run code analysis on SonarQube and SonarCloud.
+
+# How to use this image
+
+To scan your repository, you have to:
+
+-	Pass your source code to the scanner. Your source code repository should be mounted to the containers `/usr/src` directory since this is its default work directory.
+-	Define `sonar.host.url` and `sonar.projectKey` in the `sonar-project.properties` file in the root of your source code repository. The default URL is `http://localhost:9000`, but you'll most likely need to change it to fit your setup.
+-	Run the following command:
+
+```console
+docker run \
+    --rm \
+    -e SONAR_HOST_URL=http://foo.acme:9000 \
```

Comment:
I used the example from the SonarScanner docs here if we are going to be recommending using sonar-project.properties. If we recommend sonar.host.url in properties, then i'm guessing we don't need to set the env var here?

Do we still want to talk about being able to set the properties with -Dsonar.projectKey and -Dsonar.host.url or the env var?